tools/cnftParams.ts

```diff
@@ -0,0 +1,100 @@
+// import { PublicKey } from "@solana/web3.js"
+import { Connection, PublicKey } from '@solana/web3.js'
+import { ConcurrentMerkleTreeAccount } from '@solana/spl-account-compression'
+import * as bs58 from 'bs58'
+import { fetchDasAssetProofById } from '@hooks/queries/digitalAssets'
+import { getNetworkFromEndpoint } from '@utils/connection'
+import * as anchor from '@coral-xyz/anchor'
+
+class AccountData {
+  pubkey: PublicKey
+  isSigner: boolean
+  isWritable: boolean
+  constructor(
+    pubkey: PublicKey | string,
+    isSigner = false,
+    isWritable = false
+  ) {
+    this.pubkey = typeof pubkey === 'string' ? new PublicKey(pubkey) : pubkey
+    this.isSigner = isSigner
+    this.isWritable = isWritable
+  }
+}
+
+export function decode(stuff: string) {
+  return bufferToArray(bs58.decode(stuff))
+}
+
+function bufferToArray(buffer: Buffer): number[] {
+  const nums: number[] = []
+  for (let i = 0; i < buffer.length; i++) {
+    nums.push(buffer[i])
+  }
+  return nums
+}
+
+export async function getCompressedNftParamAndProof(
```

Comment:
my understanding is that this is supposed to be creating the compressed nft param for use with nft voter v2 program. a docstring comment should explain this, otherwise this file is pure mystery for anyone who isn't reviewing a PR about nft voter v2